src/Cdn/Cdn/Az.Cdn.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.1'
+ModuleVersion = '1.1.0'
```

Comment:
Please do not change this module version - we do this automatically during release processing.

Reply:
reverted